File: baekjoon/10871/10871.cpp
#include <iostream>
using namespace std;

int	main(void)
{
	int	num, max, iter;

	cin >> iter >> max;
	for (int i = 0; i < iter; i++)
	{
		cin >> num;
		if (num < max)
			cout << num << ' ';
	}
	return (0);
}

File: baekjoon/10871/10871.py
iter, max_num = map(int, input().split())
nums = list(map(int, input().split()))
for i in nums:
	if (i< max_num):
		print(i, end=' ')
  
# iter, max = map(int, input().split())
# num = list(map(int, input().split()))
# for i in range(iter):
# 	if num[i] < max:
# 		print(num[i], end=' ')